app-modules/ai/src/Enums/AiModel.php

```diff
@@ -133,6 +137,8 @@ public function getServiceClass(): string
             self::OpenAiGpt41Mini => OpenAiGpt41MiniService::class,
             self::OpenAiGpt41Nano => OpenAiGpt41NanoService::class,
             self::OpenAiGptO4Mini => OpenAiGptO4MiniService::class,
+            // We don't have a service for this yet, but we can use the test service for now
```

Comment:
What is the reason we need to add this here?

IMO, the Jina dropdown for applicable features should only include the research advisor, so we should never need to call this method. Selecting anything other than the research advisor would not work. Non-Jina models should not be able to select the research advisor as a feature, as they don't work.

As per your comment in Jira, I think a `getResearchServiceClass()` method and interface should be introduced when we add a second model to the research advisor implementation, but at the moment we should only have one option.

Reply:
@danharrin I made you first requested change but am going to create a separate ticket for the other.